packages/client-core/src/user/components/UserMenu/menus/EmoteMenu.tsx

```diff
@@ -39,7 +39,7 @@ export const useEmoteMenuHooks = ({ changeActiveMenu }: EmoteMenuHooksProps) =>
     {
       body: <img src="/static/sad.svg" alt="sad" />,
       containerProps: {
-        onClick: () => runAnimation(AvatarStates.CLAP)
+        onClick: () => runAnimation(AvatarStates.DANCE2)
```

Comment:
why this change?

Reply:
There'd been a comment about how it was confusing that the animation with the hand clapping icon wasn't the animation that actually clapped, so I swapped them. This admittedly isn't related to the other avatar fixes, I just threw it in. If there's a reason to the icon/animation pairing I overlooked, it can be reverted.

Reply:
no prob, i'll look into it